.drone.yml

```diff
@@ -746,6 +746,9 @@ steps:
   - cd ../secretsencryption
   - vagrant destroy -f
   - go test -v -timeout=30m ./secretsencryption_test.go -ci -local
+  - cd ../startup
+  - vagrant destroy -f
+  - go test -v -timeout=30m ./startup_test.go -ci -local
```

Comment:
Do we want to do this one first? Seems like it'll catch a lot of basic stuff before we even get into the more complicated tests.

Reply:
Good idea, I will change this in a future PR, to not block the release of 1.27